ASR_NL_benchmark/__main__.py

```diff
@@ -29,7 +38,12 @@
         interface.main()
     else:
         print('Running benchmarking')
-        benchmarking = pipeline.Pipeline(args.hypfile[0], args.hypfile[1], args.reffile[0], args.reffile[1], kind=args.kind)
+        skip_ref_norm = args.skip_ref_normalization
+        skip_hyp_norm = args.skip_hyp_normalization
+        if args.skip_normalization:
+            skip_ref_norm = args.skip_ref_normalization
+            skip_hyp_norm = args.skip_hyp_normalization
+        benchmarking = pipeline.Pipeline(args.hypfile[0], args.hypfile[1], args.reffile[0], args.reffile[1], kind=args.kind, skip_ref_norm=skip_ref_norm, skip_hyp_norm=skip_hyp_norm)
```

Comment:
Personally I think it's neater to pass args.skip_ref_normalization, args.skip_hyp_normalization and args.skip_normalization into pipeline.Pipeline directly (and do the if logic in pipeline.Pipeline.

Reply:
I now see why you apply the logic here, otherwise a lot of other functions need an extra argument. Maybe it is indeed better to keep it the way it is now.